src/audits/colors.js

```diff
@@ -36,31 +36,29 @@ module.exports = function ( files = [] ) {
 		),
 	];
 
-	return [
-		{
-			id: 'unique',
-			label: 'Number of unique colors',
-			value: uniqColors.length,
-		},
-		{
-			id: 'unique-opaque',
-			label: 'Number of unique colors (ignoring opacity)',
-			value: uniqOpaqueColors.length,
-		},
-		{
-			id: 'all-colors',
-			label: 'List of all colors',
-			value: uniqColors,
-		},
-		{
-			id: 'top-10-colors',
-			label: 'Top 10 most-used colors',
-			value: colorsByCount.slice( 0, 10 ),
-		},
-		{
-			id: 'bottom-10-colors',
-			label: 'Top 10 least-used colors',
-			value: colorsByCount.slice( -10 ).reverse(),
-		},
-	];
+	return {
+		audit: 'colors',
+		data: [
+			{
+				label: 'Number of unique colors',
+				value: uniqColors.length,
```

Comment:
These are missing the ID values, looks like they got lost in the merge

Reply:
Fixed! Thanks.